covid_graphs/covid_graphs/country_graph.py

```diff
@@ -56,6 +56,26 @@ def _get_display_range(
     return start_date, display_until
 
 
+# TODO: move this to another file
+def _get_earliest_displayable_idx(
+    fitted_formulas: Iterable[FittedFormula], max_peak_distance: datetime.timedelta
+) -> int:
+    peak_days = [
+        formula.get_peak(country_report=None) for formula in reversed(list(fitted_formulas))
+    ]
+
+    min_peak, max_peak = peak_days[0], peak_days[0]
+    take = 1
```

Comment:
Maybe a more descriptive name can be chosen?

Reply:
`take` is a classic functional concept, see e.g.:

- https://docs.microsoft.com/en-us/dotnet/api/system.linq.enumerable.take?view=netcore-3.1

- http://zvon.org/other/haskell/Outputprelude/take_f.html

However, I can rename it to `take_count`.

Reply:
I have not encountered it, so feel free to keep it, it was just a suggestion :)